src/ccstudiodss/api.py

```diff
@@ -64,6 +64,7 @@ class Session:
     script = attr.ib(default=None)
     debug_server = attr.ib(default=None)
     debug_session = attr.ib(default=None)
+    device_string = attr.ib(default='.*')
```

Comment:
If we keep this covering only the single-parameter `openSession(sPattern)` signature this should probably reflect the regex nature.  `device_pattern` or `device_regex` perhaps.

Reply:
device_pattern works for me